src/Our.Umbraco.InnerContent/Web/UI/App_Plugins/InnerContent/js/innercontent.controllers.js

```diff
@@ -416,9 +473,9 @@ angular.module("umbraco").factory('innerContentService', [
             return icResources.getContentTypeIconsByGuid(guids);
         }
 
-        self.createEditorModel = function (contentType, dbModel) {
+        self.createEditorModel = function (contentType, blueprintId, dbModel) {
```

Comment:
Any changes made to the publish api of innerContentService should be classed as a breaking change

Reply:
I was thinking of v2'ing this. But yes, if there's a way to not have a breaking-change, that'd be ace - I'll re-review the code.

Reply:
@mattbrailsford I've updated the PR, but not sure what to do about this one.

Reply:
If we wanna maintain backwards compat, I'd say move it to the end of the method signature, or, run some code to see if it's a valid blueprintId or a dbModel in position two, if it's a dbModel, then set the dbModel var value if you get what I mean?

Reply:
PR updated - I've moved it to the end of the method signature.  I think checking the object-type of the 2nd argument feels a bit urgh.